modules/client-common/pom.xml

```diff
@@ -95,6 +95,11 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.ignite</groupId>
+            <artifactId>ignite-calcite</artifactId>
```

Comment:
`ignite-client-common` is for sharing serialization logic between `ignite-client-handler` and `ignite-client`. We should not have such a heavy reference here, most likely it should be moved to `ignite-client-handler`.